src/Test/Perf/util/test_util.csx

```diff
@@ -17,33 +17,36 @@ using System.Net;
 // Arguments
 //
 
+// This is due to a "bug" in csi that has Args non-static.  Thus, you can't 
+// use it from non-static methods.
+static IEnumerable<string> StaticArgs = null;
```

Comment:
What coding style are we following for field names in CSX?  This file seems to be very inconsistent.